Dockerfile

```diff
@@ -1,23 +1,23 @@
 # Dockerfile
-FROM ruby:2.3
+FROM ruby:2.3-alpine
 
 ENV PORT 3000
 ENV PATH /node_modules/.bin:$PATH
 
-RUN curl -sL https://deb.nodesource.com/setup_8.x | bash - \
-  && apt-get update -qq \
-  && apt-get install -y \
-    build-essential \
-    libpq-dev \
+RUN apk update \
```

Comment:
Alpine Linux uses a different package manager called apk, so all the apt-get commands must be changed